doc/user_guide/transform/index.rst

```diff
@@ -96,6 +97,7 @@ Transform                                  Supported
 :ref:`user-guide-bin-transform`              ✔
 :ref:`user-guide-calculate-transform`        ✔
 :ref:`user-guide-density-transform`
+:ref:`user-guide-extent-transform`
```

Comment:
@jonmmease Does VegaFusion support the extent transform? The `.transformed_data` method worked on the chart example I built in the doc page but as I use the parameter value as single values it does not return it anyway so not sure if that's a good test.

Reply:
Yes, VegaFusion supports extent, so I think we're good there

Reply:
Cool, I'll add the tick in the docs